<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>establishdialog</class>
 <widget class="QDialog" name="establishdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/establish.png</normaloff>:/images/establish.png</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget
{
	background-image: url(:/images/loginbackground.jpeg);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_25">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 24pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>创建域名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QPushButton" name="peopleButton">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>20</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: #1fab89;
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 10pt;
	font-weight: bold;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>创建</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;人工输入&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;无法输入不合法的顶级域名。&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;已经存在的域名不会重新被插入。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QtMaterialAutoComplete" name="searchLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>人工输入</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="randomButton">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>20</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: #1fab89;
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 10pt;
	font-weight: bold;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>创建</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;随机输入&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;选择等级和每节域名长度，随机生成域名&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;仅供测试使用&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QSpinBox" name="spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>4</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="lengthBox">
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>8</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; 文件输入&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; 选择txt文件以直接导入大量的数据。&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; 每个域名占用一行即可。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="fileButton">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>20</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: #1fab89;
	min-width: 80px;
	font-family: &quot;Times New Roman&quot;;
	font-size: 10pt;
	font-weight: bold;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>创建</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialAutoComplete</class>
   <extends>QLineEdit</extends>
   <header location="global">qtmaterialautocomplete.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
